dbms/src/main/resources/log4j2.xml

```diff
@@ -44,6 +44,10 @@
         </logger>
         <!-- <logger name="ch.unibas.dmi.dbis.polyphenydb.jdbc" level="TRACE">
         </logger> -->
+        <!--<logger name="ch.unibas.dmi.dbis.polyphenydb.plan.RelOptPlanner" level="DEBUG">
+        </logger> -->
+        <!-- <logger name="ch.unibas.dmi.dbis.polyphenydb.plan.hep.HepPlanner" level="TRACE">
+        </logger> -->
```

Comment:
Why are there commented out lines added here?

Reply:
Because this is helpful for debugging the planner and optimizer. Leaving them in the file allows to easily enable them if required.